<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModePasPaswidget</class>
 <widget class="QWidget" name="ModePasPaswidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_chandelier">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>671</width>
     <height>331</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>202</width>
     <height>518</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="VL_menu">
    <item>
     <layout class="QVBoxLayout" name="VL_menuFichier">
      <item>
       <widget class="QPushButton" name="pushButton_retourMenu">
        <property name="text">
         <string>Retour au menu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_sauvegarder">
        <property name="text">
         <string>Sauvegarder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_parametre">
        <property name="text">
         <string>Parametres</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_aide">
        <property name="text">
         <string>Aide</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="VSpacer_menuFichierTimer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>35</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_timer">
      <item>
       <property name="text">
        <string>Pause</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Avancer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Reculer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DÃ©filer</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="VSpacer_TimerMenuTransaction">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>35</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="VL_menuTransaction">
      <item>
       <layout class="QHBoxLayout" name="HL_infoMontant">
        <item>
         <widget class="QLabel" name="label_montantBase">
          <property name="text">
           <string>MontantB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_montantContrepartie">
          <property name="text">
           <string>MontantC</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_broker">
          <property name="text">
           <string>Broker</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="HL_infoOpen">
        <item>
         <widget class="QLabel" name="label_date">
          <property name="text">
           <string>Date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_openPrice">
          <property name="text">
           <string>OpenPrice</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_montant"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="HL_achatvente">
        <item>
         <widget class="QPushButton" name="pushButton_achat">
          <property name="text">
           <string>Achat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_vente">
          <property name="text">
           <string>Vente</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_annulerTransaction">
        <property name="text">
         <string>Annuler transaction</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="VSpacer_menuTransactionMenuOption">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>35</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="VL_menuOptions">
      <item>
       <widget class="QPushButton" name="pushButton_tabCoursOHLCV">
        <property name="text">
         <string>Tableau des cours</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_tabTransaction">
        <property name="text">
         <string>Tableau des transactions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_prendreNote">
        <property name="text">
         <string>Prendre des notes</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="tabWidget_autres">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>671</width>
     <height>141</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabsClosable">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab_RSI">
    <attribute name="title">
     <string>RSI</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_coursOHLVC">
    <attribute name="title">
     <string>coursOHLCV</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>186</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="HL_menuIndicateur">
    <item>
     <widget class="QCheckBox" name="checkBox_RSI">
      <property name="text">
       <string>RSI</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_EMA">
      <property name="text">
       <string>EMA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_MACD">
      <property name="text">
       <string>MACD</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
